src/src/PluginStructs/P128_data_struct.h

```diff
@@ -2241,7 +2241,7 @@ const uint8_t PROGMEM ftv_gamma8[] = {
 
 const float pi = 3.1415926535897932384626433832795;
```

Comment:
A float can only hold 6 - 7 decimals. Also this number is first a double, so you can add a `f` to it.
But I think Arduino has some define for `M_PI` or something like that.

Reply:
Yeah, that's the 'trouble' with migrating/inheriting code, it was already there 😃 
I'll fix 👍

Reply:
Worst is, it's not even used in the code... 😆 removed.